mautic/mautic
# Report export with aggregated AVG columns are converted to integers

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ Y ]
| New feature/enhancement? (use the a.x branch)      | [ N ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Report export to CSV, XLS and HTML with aggregated AVG columns is not correct, AVG values are converted to integers. This PR fixes this issue.



#### Steps to replicate this issue:


1. Create test contacts with points and assign them to countries (in next step we will calculate average country contact points)
2. Create report with **Contact** datasource with following settings:
![obraz](https://user-images.githubusercontent.com/8580942/192505826-a41caf0f-1cc2-423d-8619-ac22d63cb451.png)
3. Save report & close. Report should look like this:
![obraz](https://user-images.githubusercontent.com/8580942/192506273-722161e0-27cb-42dd-a2f2-4d494b6c47ec.png)
4. Try different export types, result should look like this:
![obraz](https://user-images.githubusercontent.com/8580942/192506549-ef67cfb2-4d59-4a40-8f94-447d2e13f3ed.png)
Note the AVG Points value is rounded to integer. It's the same for CSV, XLS and HTML export.

#### Steps to test this PR:
1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Repeat steps 1-3 from steps to replicate
3. Export result should be correct:
![obraz](https://user-images.githubusercontent.com/8580942/192507349-0b028407-530e-47a3-84d8-33c186b59207.png)
